modules/search-syntax.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * operating/search-filter.adoc
+:_module-type: CONCEPT
+[id="search-syntax_{context}"]
+= Search syntax
+
+A search query is made up of two parts:
+. An attribute that identifies the resource type you are after.
```

Comment:
* Probably should be an unnumbered list?
* "you are after" sounds a little casual. Could it be reworded, like "you want to search for" or something similar?